pages/signIn.tsx

```diff
@@ -14,19 +15,37 @@ const callSignInApi = async () => {
   console.log(await data.json())
 }
 
-
 function signUp(props) {
+  const [email, setEmail] = useState()
+  const [password, setPassword] = useState()
+  const [toolTip, setToolTip] = useState(false)
+  const [type, setType] = useState('')
+
+function handleEmailInput(event){  
+  setEmail(event.target.value)
+
+}
+
+function handlePasswordInput(event){
```

Comment:
[pebble] I do not like the fact that  `handlePasswordInput` and `handleEmailInput` do basically the same thing, just for different states. Instead try to make a function that takes in the event, and the state you should be setting

Reply:
done, helper functions bae beee